src/Command/CommandBase.php

```diff
@@ -1023,6 +1025,71 @@ protected function fillMissingRequiredApplicationUuid(InputInterface $input, Out
     }
   }
 
+  /**
+   * @param $userUuidArgument
+   * @param $orgUuidArgument
+   */
+  protected function convertUserAliasToUuid(InputInterface $input, $userUuidArgument, $orgUuidArgument): void {
+    if ($input->hasArgument($userUuidArgument)
+      && $input->getArgument($userUuidArgument)
+      && $input->hasArgument($orgUuidArgument)
+      && $input->getArgument($orgUuidArgument)
+    ) {
+      $userUuID = $input->getArgument($userUuidArgument);
+      $orgUuid = $input->getArgument($orgUuidArgument);
+      $userUuid = $this->validateUserUuid($userUuID, $orgUuid);
+      $input->setArgument($userUuidArgument, $userUuid);
+    }
+  }
+
+  /**
+   * @param String $userAlias
+   *   User alias like uuid or email.
+   * @param String $orgUuidArgument
+   *   Organization uuid.
+   * @return string
+   *   User uuid from alias
+   */
+  protected function validateUserUuid($userUuidArgument, $orgUuidArgument): mixed {
+    try {
+      self::validateUuid($userUuidArgument);
+    }
+    catch (ValidatorException) {
+      // Since this isn't a valid UUID, assuming this is email address.
+      return $this->getUserUuidFromUserAlias($userUuidArgument, $orgUuidArgument);
+    }
+
+    return $userUuidArgument;
+  }
+
+  /**
+   * @param String $userAlias
+   *   User alias like uuid or email.
+   * @param String $orgUuidArgument
+   *   Organization uuid.
+   * @return string
+   *   User uuid from alias
+   */
+  protected function getUserUuidFromUserAlias(String $userAlias, String $orgUuidArgument): string {
+    $acquiaCloudClient = $this->cloudApiClientService->getClient();
+    $organization_resource = new Organizations($acquiaCloudClient);
+    $org_members = $organization_resource->getMembers($orgUuidArgument);
```

Comment:
Organization api also separate method to get the admins. For now, added only members.
If required, we can also get admins and merge the list to check from both